docs/api/web-contents.md

```diff
@@ -874,20 +874,47 @@ Returns:
 * `event` Event
 * `dirtyRect` [Rectangle](structures/rectangle.md)
 * `image` [NativeImage](native-image.md) - The image data of the whole frame.
+* `texture` [OffscreenSharedTexture](structures/offscreen-shared-texture.md) - The GPU shared texture of the frame. Will be `null` if `webPreferences.offscreenUseSharedTexture` is `false`.
 
 Emitted when a new frame is generated. Only the dirty area is passed in the
 buffer.
 
 ```js
 const { BrowserWindow } = require('electron')
 
-const win = new BrowserWindow({ webPreferences: { offscreen: true } })
-win.webContents.on('paint', (event, dirty, image) => {
-  // updateBitmap(dirty, image.getBitmap())
+const win = new BrowserWindow({ webPreferences: { offscreen: true, offscreenUseSharedTexture: true } })
+win.webContents.on('paint', (event, dirty, image, texture) => {
+  if (texture) {
+    // importTextureHandle(dirty, texture.textureInfo)
```

Comment:
It would be great to have a little more info on how an app can actually use this. Perhaps in a separate guide doc?

Reply:
It is currently very user dependent, personally I use a spout plugin [here](https://github.com/reitowo/electron-spout) to share the texture to external processes. So I think it maybe difficult to provide an pure electron example?
But good news is, it's possible to make WebGPU support importing such texture (with future works). However we probably need to find out how to pass `PendingRemote` to arbitrary processes as it needs to be imported at destination process. (Or not, you can always just pass the `textureInfo` and let the original process know to call the release).